src/react-components/presence-log.js

```diff
@@ -4,11 +4,56 @@ import styles from "../assets/stylesheets/presence-log.scss";
 import classNames from "classnames";
 import { FormattedMessage } from "react-intl";
 import ChatMessage from "./chat-message";
+import { share } from "../utils/share";
+
+function SpawnPhotoMessage({ name, body: { src: url }, className, maySpawn, hubId }) {
+  let landingPageUrl = new URL(url);
+  const [hostname, port] = process.env.RETICULUM_SERVER.split(":");
+  console.log(hostname, port, landingPageUrl.port);
```

Comment:
console log

Reply:
I might consider rolling this routine into `phoenix-utils.js`, its where I've tried to centralize references to `RETICULUM_SERVER`

Reply:
any reason `getReticulumFetchUrl(landingPageUrl.pathname.replace(".png", ".html"))` wouldn't work? also might be worth future proofing this for other file extensions by just dropping the file extension instead of assumign `.png`